### ISSUE TITLE
Remove hardcoded L1ChainIds values

### ISSUE THREAD

**Issue by gndelia** [hemilabs/ui-monorepo#376]
As part of enabling Bitcoin deposits (See #342 and #345), we had to add the `chainId` field to the Transaction History stored in local Storage. This field wasn't part of it originally, so reading that data would cause an error if the code assumes it was defined. As a fix for that was implemented in TBD PR, we left a series of defaults when this value was defined. As just entering the Tx history for every user should automatically fix this problem (saving all deposits + withdrawals with the corresponding chainId), we can remove these defaults in a few weeks, or in the worst case, just pre-launch.

TL;DR remove hardcoded default chainIds where this issue was referenced
